templates/tfengine/components/cicd/variables.tf

```diff
@@ -64,3 +64,9 @@ variable "build_viewers" {
   description = "List of IAM members to grant cloudbuild.builds.viewer role in the devops project to see CICD results"
   default     = []
 }
+
+variable "fail_on_deletes" {
```

Comment:
Maybe a better name is reject_deletions_in_plan?

Reply:
Done. Changed to "reject_plan_deletions".